### DISCUSSION TITLE
Shop items

### DISCUSSION THREAD

**Discussion by Milo123459** [Milo123459/Spencer#114] [Ideas]
👋 Hello!

You can comment on this thread to suggest an item for the shop, please leave a name and an indepth description.

Thanks 😄

**Comment by comic-dev** [comment]
Maybe a coin/xp multiplier?

**Reply by Milo123459** [reply]
Planned!

**Reply by comic-dev** [reply]
pog 🎉

**Comment by Pazdikan** [comment]
pink phallic object

**Reply by Milo123459** [reply]
😳

**Reply by comic-dev** [reply]
:point:

**Reply by CosmicDev360** [reply]
no

**Comment by mistyolive** [comment]
make a booster that cuts your waiting time

**Reply by mistyolive** [reply]
when are trying to rob someone (salvage)

**Reply by Milo123459** [reply]
Good idea! It'll have to be expensive.
